src/tv2-common/actions/executeAction.ts

```diff
@@ -1211,12 +1206,32 @@ async function executeActionCutToCamera<
 			...(settings.SourceLayers.EVS ? [settings.SourceLayers.EVS] : []),
 			settings.SourceLayers.Continuity
 		])
+		await stopGraphicPiecesThatShouldEndWithPart(context, currentPieceInstances)
 
 		kamPiece.enable = { start: 'now' }
 		await context.insertPiece('current', kamPiece)
 	}
 }
 
+async function stopGraphicPiecesThatShouldEndWithPart(
+	context: ITV2ActionExecutionContext,
+	currentPieceInstances: Array<IBlueprintPieceInstance<unknown>>
+) {
+	await context.stopPieceInstances(
+		currentPieceInstances
+			.filter(pieceInstance => isGraphicThatShouldEndWithPart(pieceInstance))
+			.map(pieceInstance => pieceInstance._id)
+	)
+}
+
+function isGraphicThatShouldEndWithPart(pieceInstance: IBlueprintPieceInstance<unknown>): unknown {
```

Comment:
This methods name is indicating that it should return a boolean, but it returns unknown?

Reply:
I think that came from automatic refactoring and I didn't notice :P